ldms/src/sampler/llnl/lustre_client.c

```diff
@@ -0,0 +1,654 @@
+/* -*- c-basic-offset: 8 -*-
+ * Copyright (c) 2019 Lawrence Livermore National Security, LLC.
+ * Produced at the Lawrence Livermore National Laboratory. Written by
+ * Christopher J. Morrone <morrone2@llnl.gov>
+ * All rights reserved.
+ *
+ * This work was performed under the auspices of the U.S. Department of
+ * Energy by Lawrence Livermore National Laboratory under
+ * Contract DE-AC52-07NA27344.
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the BSD-type
+ * license below:
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions
+ * are met:
+ *
+ *      Redistributions of source code must retain the above copyright
+ *      notice, this list of conditions and the following disclaimer.
+ *
+ *      Redistributions in binary form must reproduce the above
+ *      copyright notice, this list of conditions and the following
+ *      disclaimer in the documentation and/or other materials provided
+ *      with the distribution.
+ *
+ *      Neither the name of Sandia nor the names of any contributors may
```

Comment:
This should probably say "Neither the name of  Lawrence Livermore ... ", not Sandia..

Reply:
@valleydlr  owes us some verbiage here more generally. apparently there was some concern with snl legal.

Reply:
I made a github pr covering the license issue.